src/fetcher.ts

```diff
@@ -80,12 +80,6 @@ export function createFetchComponent(defaultOptions?: FetcherOptions): IFetchCom
       abortController: controller
     })
 
-    // Throw in case of error
-    if (!response?.ok) {
-      const responseText = await response?.text()
-      throw new Error(`Failed to fetch ${url}. Got status ${response?.status}. Response was '${responseText}'`)
-    }
-
     if (!isUsingNode() && !!response) {
```

Comment:
```suggestion
    if (!isUsingNode() && !!response && response?.ok) {
```
do we need to add `response?.ok` here so it creates the buffer just if `response.ok` is truthy?

Reply:
yeah, true, they should return the buffer as well. LGTM then!!